extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -124,7 +124,7 @@ angular.module('oppia').component('svgFilenameEditor', {
         bold: false,
         italic: false
       };
-      ctrl.enableRemoveButton = false;
+      ctrl.isObjectSelected = false;
```

Comment:
Use objectIsSelected instead. We generally prefix `is` to functions.

Reply:
Done.